### ISSUE TITLE
Pug in Deneb before free worlds

### ISSUE THREAD

**Issue by McloughlinGuy** [endless-sky/endless-sky#5291]
**Is your feature request related to a problem? Please describe.**
I noticed that if you go to Deneb before the free worlds storyline, there are out all over the place constantly using there jump drives. Where do they go? Not once do you encounter a Pug ship in the systems nearby, yet the Pug seem to be jumping all over the place. It seems impossible that no one would notice this.

**Describe the solution you'd like**
I would like the Pug ships to not make any hyperspace jumps while in the system, seeing as there isn’t anywhere they can go without attracting a ton of attention

**Describe alternatives you've considered**
An alternative would be to just remove them entirely until they begin the invasion, but that would be unethical.

**Additional context**
The way to get to Deneb before free worlds is to take a jump drive from a Korath raider, though most of you probably already know that.

**Comment by Terin** [comment]
The ships rapidly jumping in and out, without any jump drives is very strange, yeah, especially knowing they'd appear immediately in some other system in full view. Perhaps pre-conflict Pug ships could have cloaking built into them? Does that behavior even work?

Removing the ships could work as well, as they don't have much need to go scouting or raid for resources; they seem fairly omniscient as-is. And granted, finding Deneb before getting the jump drive in the Free Worlds campaign is pretty rare for newer players, but the system is there to be found. If it wasn't supposed to be seen before the Pug act up, it could just not exist at first, and be added with an event.

**Comment by Zitchas** [comment]
Given that the Pug are actually Tier ? (where ? is greater than 3), I think it is generally assumed that when the Pug jump out of Deneb and *don't* appear in any of the surrounding systems, they are actually going somewhere we can't follow. Perhaps they know of an alternate way to configure their JD so that when they disappear, they are actually going somewhere else. Perhaps jumping directly into a wormhole? Maybe into another dimension? Maybe there are other systems nearby that don't register as hyperjump destinations? The point is, we don't know. Maybe they aren't actually jumping anywhere, but we just can't tell the difference between a jump drive effect and some sort of ground-based teleporter?

Alternatively, they may well all be just giant super-advanced holodeck-style projections - things that are physically there, but cease to exist when they do their "jump-out" sequence. 

We have about as good a chance of knowing where and how they are travelling as humans from the 16th century would have watching a jet take off. It is worth noting that the basic hyperdrive is so advanced that the description states:
> "They say that fewer than a dozen human beings alive today understand the inner workings of the hyperdrive."

Likewise, the description for the jump drive states:
> "The workings of the jump drive are a total mystery. The best that human scientists have been able to find out is that this device is a containment field for matter in a state that cannot normally exist in this universe; if the drive is opened up, the field collapses and its contents disappear. Therefore, even though a few jump drives have been captured over the years, it has so far proved impossible to reverse engineer them."

Long story short: We don't know all the capabilities of the Pug or Jump Drives. Both quite possibly have a *lot* of capabilities  that we don't know about.

**Comment by McloughlinGuy** [comment]
Yes, but they don’t have anything which allows them to travel where humans can’t reach. Maybe add something to allow them to do that, but make the ships harder to capture?

**Comment by McloughlinGuy** [comment]
After I attacked the pug in Deneb, one of the ships followed me into Altair. It wasn't cloaked or anything. There is no possible way that nobody noticed that.

**Comment by Zitchas** [comment]
To be fair, no-one notices when I fight in the Free Worlds navy (and even become a respected officer) flying a giant semi-organic spaceship with a fleet of alien shield-shaped ships that wipes the floor with anything the Republic navy can field against me. So, right now, as things stands, no-one bats an eye at strange looking ships.

Personally, I think that there is a huge DIY / custom ship market that we don't see that is filled with weird looking ships, much like how these days there are people making all sorts of custom cars and aircraft that look really weird. All we see are the ones with manufacturing lines. (This is also, as far as that goes, what my headcanon is that all the person ships are. Custom ships made by DIY and/or rich people)

**Comment by McloughlinGuy** [comment]
Wouldn’t using a jump drive create some suspicion? I mean, it’s not like the Quarg have DIY ships.

**Comment by Plip-Plop** [comment]
Maybe instead of using their jump drive, they could jump through a wormhole only Pug ships can use. Maybe make the wormhole already in Deneb send ships to a different place depending on what special outfits they have.

**Comment by McloughlinGuy** [comment]
This one will be left open, for now.

**Comment by McloughlinGuy** [comment]
I don’t care about this anymore. Pug in Deneb can be used for farming jump drives easily.
